packages/millicast-sdk/package-lock.json

```diff
@@ -1,14 +1,16 @@
 {
   "name": "@millicast/sdk",
-  "version": "0.1.44-rc-1",
+  "version": "0.1.44",
   "lockfileVersion": 2,
   "requires": true,
   "packages": {
     "": {
       "name": "@millicast/sdk",
-      "version": "0.1.44-rc-1",
+      "version": "0.1.44",
       "license": "See in LICENSE file",
       "dependencies": {
+        "@dolbyio/webrtc-stats": "^0.4.0",
+        "@peermetrics/webrtc-stats": "^5.7.1",
```

Comment:
Just wanting to be sure - do we need `peermetrics` as well?

Reply:
Huh, I thought I removed that.  Not sure how it got there.